CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Unreleased
 
+- Fix documentation building on `docs.rs`.
```

Comment:
Not a user-facing change, probably not useful to have in the changelog - maybe just yank `0.4.7` given the nature of the `.8` followup?